library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Nilesh, you are the formally trained programmer here.  I am just an amateur enthusiast.  Do it that way if you like it, but like Terry said, you are in charge of this.  We are only here to whine if it breaks! 😈